docs/deployment/on_kubernetes.md

```diff
@@ -5,3 +5,111 @@
 </div>
 
 # Running Kyuubi on Kubernetes
+
+## Requirements
+
+When you want to run Kyuubi's Spark SQL engines on Kubernetes, you'd better have cognition upon the following things.
+
+* Read about [Running Spark On Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)
+* An active Kubernetes Cluster
+* Kubectl
+* KubeConfig of the target cluster
+
+## Configurations
+
+### Test Cluster
+
+You can use the shell code to test your cluster whether it is normal or not.
+
+```shell
+$SPARK_HOME/bin/spark-submit \
+ --master K8s://https://<K8s-apiserver-host>:<K8s-apiserver-port> \
+ --class org.apache.spark.examples.SparkPi \
+ --conf spark.executor.instances=5 \
+ --conf spark.dynamicAllocation.enabled=false \
+ --conf spark.shuffle.service.enabled=false \
+ --conf spark.kubernetes.container.image=<spark-image> \
+ local://<path_to_examples.jar>
+```
+
+When running shell, you can use cmd `kubectl describe pod <podName>` to check if the information meets expectations.
+
+### Master
+
+Spark on Kubernetes config master by using a special format.
+
+`K8s://https://<K8s-apiserver-host>:<K8s-apiserver-port>`
+
+You can use cmd `kubectl cluster-info` to get api-server host and port.
+
+### Spark Image
+
+Kubernetes requires users to supply images that can be deployed into containers within pods.
+
+Spark also ships within a `./bin/docker-image-tool.sh` script that can be used to build and publish the Docker images to use
```

Comment:
do we need to build the image and get the cluster-info first before telling users to run the example code?

Reply:
Will adjust Master and image before test